### PR TITLE
add IE11 polyfill to support URI download scheme

### PR DESCRIPTION
**Pull request by HiroyasuNishiyama**
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

As described in this [issue](https://github.com/node-red/node-red/issues/2755), exporting flow do not work on IE11.
This is caused by use of data uri scheme for downloading.
This PR try to fix this problem by adding polyfill for supporting this feature.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: packages/node_modules/@node-red/editor-client/src/js/polyfills.js [hunk 1]
```diff
@@ -53,5 +53,31 @@
             Set.prototype.constructor = Set;
         }
 
+        if (window.navigator.msSaveBlob) {
+            // IE does not support data uri scheme for downloading data
+            window.addEventListener("click", function (ev) {
```
Comment (knolleary) [inline packages/node_modules/@node-red/editor-client/src/js/polyfills.js:58]: Adding an event handler on every single click on the page feels like a lot of overhead to fix one button in the UI.

Can we make this a utility function that we call ourselves from the click handler of the download buttons?

**Review by knolleary — CHANGES_REQUESTED** [review]

File: packages/node_modules/@node-red/editor-client/src/js/polyfills.js (hunk 1 above)
Comment (HiroyasuNishiyama) [reply packages/node_modules/@node-red/editor-client/src/js/polyfills.js]: Hi. Moved download code to a separate function.

**Comment by knolleary** [comment]
For future reference, bug fixes should target the master branch so they can be released in the maintenance releases.

**Comment by HiroyasuNishiyama** [comment]
Ah, yes.  It should be targeted to master.

**Pull request merged**